content/en/docs/reference/access-authn-authz/extensible-admission-controllers.md

```diff
@@ -111,18 +119,33 @@ webhooks:
     - CREATE
     resources:
     - pods
+    scope: "Namespaced"
   clientConfig:
     service:
       namespace: <namespace of the front-end service>
       name: <name of the front-end service>
+      port: 443
     caBundle: <pem encoded ca cert that signs the server cert used by the webhook>
+  admissionReviewVersions:
+  - v1beta1
+  timeoutSeconds: 1
 ```
 
+The scope field specifies if only cluster-scoped resources ("Cluster") or namespace-scoped resources
+will match this rule. "*" means that there are no scope restrictions.
+
 {{< note >}}
 When using `clientConfig.service`, the server cert must be valid for
 `<svc_name>.<svc_namespace>.svc`.
 {{< /note >}}
 
+{{< note >}}
+Default timeout for a webhook call is 30 seconds but starting kubernetes 1.14 you
+can set the timeout and it is encouraged to use a very small timeout for webhooks.
+If the webhook call timed out, it will be ignored or the API call will fail based
```

Comment:
"If the webhook call times out, the request is handled according to the webhook's failure policy."